src/main/flight/pid.c

```diff
@@ -312,16 +315,24 @@ void pidUpdateAntiGravityThrottleFilter(float throttle)
         const float antiGravityThrottleLpf = pt1FilterApply(&pidRuntime.antiGravityThrottleLpf, throttle);
         // focus P boost on low throttle range only
         if (throttle < 0.5f) {
-            pidRuntime.antiGravityPBoost = 0.5f - throttle;
+            pidRuntime.antiGravityPBoost = (0.5f - throttle)
+                * (float)(currentPidProfile->itermAcceleratorPboostMultiplier) / 100.0f;
```

Comment:
should be moved to line 900 as you can save a multiply, itermAcceleratorPboostMultiplier should also have a runtime variant so that a / 100.0 does not need to be calculated every pidloop.

Reply:
not sure why line 900.
line 900 is within a PID loop.
Probably both "runtime variables" should be just inside pidRuntime->... and initialized in pid_init